pkg/agent/openflow/pipeline.go

```diff
@@ -333,6 +344,17 @@ func (c *client) connectionTrackFlows(category cookie.Category) (flows []binding
 	return
 }
 
+// Connection track flows for traceflow
+func (c *client) traceflowConnectionTrackFlows(dataplaneTag uint8, category cookie.Category) binding.Flow {
+	connectionTrackStateTable := c.pipeline[conntrackStateTable]
```

Comment:
I think manually copying the actions might introduce errors if we update the original flow actions but forget to modify here.

@Dyanngg has added a func in OpenFlow control interface to duplicate existing flow entry to another one. You could add "Todo" here to leverage his new API in future to copy target flows with actions in a simpler way.

Reply:
Do you mean the cookie may be changed?
This function will be called when every traceflow starting, and flow will be timed out after 300 sec.

Reply:
No, not updating cookie. It is a function to copy the complete Flow to a Builder, including the match and actions. I prefer  to use that function is because I don't want you to copy action codes in the builder, otherwise if we have changed the actions in the original flow, but forget to change these "copied" builders, the flows are going different, and it is not as expected.
But it is not urgent to change in this PR, I think you can add a "TODO" here to call it in future after that function is merged.

Reply:
Got it. But I think the issue is how to get the original flow. Does this have a cache like NetworkPolicy?
Another method is combine the two functions into one, and control it by a new boolean parameter.

Reply:
Added TODO here.

Reply:
> Got it. But I think the issue is how to get the original flow. Does this have a cache like NetworkPolicy?
> Another method is integrate the two functions into one, and control it by a new boolean parameter.

Yes, actually we have cache for all OpenFlow entries, and this cache is also helpful for reconcile cases.